app/ledger.js

```diff
@@ -383,6 +391,9 @@ eventStore.addChangeListener(() => {
 var initialize = (onoff) => {
   enable(onoff)
 
+  // Check if relevant browser notifications should be shown every 15 minutes
+  setInterval(showNotifications, 15 * msecs.minute)
```

Comment:
thanks good call– it was originally there but i removed it (oops).
i added it back